packages/worker/src/utils/token.ts

```diff
@@ -0,0 +1,6 @@
+import { utils } from "zksync-web3";
```

Comment:
It seems that this function is not used in `Worker`, please remove if so.

Reply:
Done.